### ISSUE TITLE
unused argument (footer = dashboardFooter())

### ISSUE THREAD

**Issue by BirongZhang** [RinteRface/shinydashboardPlus#172]
Dear all, 

Thanks for developing such an useful tool!

When I was running the example code:
 ```
shinyApp(
    ui = dashboardPage(
      header = dashboardHeader(),
      sidebar = dashboardSidebar(),
      body = dashboardBody(),
      footer = dashboardFooter(
        left = "By Divad Nojnarg",
        right = "Zurich, 2019"
      ),
      title = "DashboardPage"
    ),
    server = function(input, output) { }
  )
```

I encountered some problems:
**Error in dashboardPage(header = dashboardHeader(), sidebar = dashboardSidebar(),  : 
  unused argument (footer = dashboardFooter(left = "By Divad Nojnarg", right = "Zurich, 2019"))**

But yesterday it worked well, today it could not.

Any advices would be highly appreciated! Thanks!

Kind regards,
Birong

**Comment by BirongZhang** [comment]
Dear all,

I guess it seems to be a compatibility issue. I exited R, re-library some packages, and now it works.

**Comment by ismirsehregal** [comment]
I guess you were using `shinydashboard::dashboardPage` instead of `shinydashboardPlus::dashboardPage`.

**Comment by HugoGit39** [comment]
What do you mean with re-library? Did  you deleted all packes and re-installed it? Cause I have the same issue

**Comment by BirongZhang** [comment]
No. I just quit R, then reopen it and library() those packages again.

**Comment by BirongZhang** [comment]
> What do you mean with re-library? Did you deleted all packes and re-installed it? Cause I have the same issue using bs4Dash which adds extra feautures, like footer but somehow I get the unused argument error with footer, but also controlbar etc

No. I just quit R, then reopen it and library() those packages again.

**Comment by HugoGit39** [comment]
Im really confused..I did this to. Killed R, re-library it...still same error

**Comment by BirongZhang** [comment]
> 

Can you run this one?
```
library(shiny)
library(shinydashboard)
library(shinydashboardPlus)

shinyApp(
   ui = dashboardPage(
     header = dashboardHeader(),
     sidebar = dashboardSidebar(),
     body = dashboardBody(),
     footer = dashboardFooter(
       left = "By Divad Nojnarg",
       right = "Zurich, 2019"
     ),
     title = "DashboardPage"
   ),
   server = function(input, output) { }
 )
```

**Comment by HugoGit39** [comment]
Wait, my bad, indeed I forgot to add. Yes works! Also bs4Dash now!

**Comment by HugoGit39** [comment]
But any knowledge about the theory behind it? Cause I guess I worked too long in the same project, though I always thought to kill R entirely bij closing using X

**Comment by ismirsehregal** [comment]
As shown in my [earlier response](https://github.com/RinteRface/shinydashboardPlus/issues/172#issuecomment-1294034872) , just use the double-colon operator ?`::` to avoid namespace issues.

**Comment by HugoGit39** [comment]
No I did that with bs4Dash:: but didnt work...I really needed to kill R and re-lib all the packages
